bin/Data/UI/EditorSpawnWindow.xml

```diff
@@ -239,7 +239,21 @@
 						<attribute name="Top Right Color" value="0.85 0.85 0.85 1" />
 						<attribute name="Bottom Left Color" value="0.85 0.85 0.85 1" />
 						<attribute name="Bottom Right Color" value="0.85 0.85 0.85 1" />
-						<attribute name="Text" value="Spawn on selected node" />
+						<attribute name="Text" value="Spawn on selected node's surface" />
```

Comment:
A object is always placed on some surface

Reply:
Not always, since this option ensures that *randomized* locations are always on the surface. Without it, having a large random radius could leave some objects hanging off the edge or not spawn at all.